src/migration/1700661785471-PostCollectionSources.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class PostCollectionSources1700661785471 implements MigrationInterface {
+    name = 'PostCollectionSources1700661785471'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "post" ADD "collectionSources" text array DEFAULT '{}'`);
```

Comment:
default should be null, isn't it?

Reply:
This is what typeorm pipes out and how we did other array fields.

Reply:
ok